### DISCUSSION TITLE
Will the Renovate CE also require storing secrets in the WebApp on Mend?

### DISCUSSION THREAD

**Discussion by dkr91** [mend/renovate-ce-ee#553] [Q&A] [answered]
Hey, will Renovate CE deprecated storing the secrets in Kubernetes or also require to store them in the WebApp as it is for Renovate itself?
https://github.com/renovatebot/renovate/pull/30132

**Comment by nabeelsaabna** [comment, marked as answer]
The Mend web app secrets is only used for the hosted app.
There is no relation between the self hosted solutions (CLI, Community and EE) secrets and the Mend web app secrets
